check_lazy.py

```diff
@@ -61,7 +61,7 @@ def sweep_models(output_filename, tests=['eval', 'train']):
                     print(f"{datetime.strptime(dt,'%Y-%m-%d %H:%M:%S')} : Running launch_command {' '.join(launch_command)}")
                     rc = subprocess.call(launch_command,
                                         env=env,
-                                        timeout = 1800, # 30 minutes, 10 min max per iter
+                                        timeout = 120, # 30 minutes, 10 min max per iter
```

Comment:
You didn't update the comment. Can we tell if the model times out or not in the artifacts?